internal/cli/atlas/deployments/options/deployment_opts.go

```diff
@@ -33,7 +33,7 @@ const (
 	spinnerSpeed         = 100 * time.Millisecond
 	// based on https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Clusters/operation/createCluster
 	clusterNamePattern    = "^[a-zA-Z0-9][a-zA-Z0-9-]*$"
-	MongotDockerImageName = "docker.io/mongodb/apix_test:mongot-preview"
+	MongotDockerImageName = "docker.io/mongodb/mongodb-atlas-search:latest"
```

Comment:
is it safe to get always the latest version? should we ping to a version to not get breaking changes?

Reply:
for now we only have one version `preview`, once atlas search takes over we might know more